.github/workflows/ci.yml

```diff
@@ -13,7 +13,7 @@ jobs:
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        os: [ubuntu-latest, macOS-latest]
+        os: [ubuntu-latest, macOS-latest, windows-latest]
```

Comment:
As you've seen, the Windows environments can be a tricky mess. I would save trying to get them integrated in for a separate specific PR when you have plenty of time to deal with headache.

Reply:
Yeah, thought this was specifically to test label position across systems. It seems to be tricky and kind of hard to debug rn. As of now this is a tracking PR
![image](https://user-images.githubusercontent.com/13226500/69270181-18f70d80-0b98-11ea-94ef-c2c18323b228.png)

Reply:
> It seems to be tricky and kind of hard to debug rn.

Yeah, seriously. :/ Good (bug) hunting to you.